robot/basestation/static/css/themes/lofi.css

```diff
@@ -0,0 +1,195 @@
+/*
+Bootstrap Buttons (Purple): #445B7B
+Regular Buttons (Green): #3D7F7F
+Table (Red): #C34641
+Table (Light Grey): #EFEFEF
+Navbar (Salmon): #A38282
+Text (Off-White): #E4E1D1
+Background (Grey): #2B343B
+Button Hover (Blueish Light Grey): #BCD4DD
+*/
+
+/* Background color */
+html, body {
+  height: 100%;
+  background-color: #2B343B;
+}
+
+/* Navigation bar color */
+.navbar {
+  background-color: #A38282;
+}
+
+/* Whole lotta bootstrap dropdown menu styling */
+.dropdown .nav-link {color:#445B7B; text-decoration: none;}
+.dropdown .btn {background: #445B7B; color:#fff;}
+.container .dropdown .dropdown-menu a:hover
+{
+  color: #fff;
+  background-color: #fff;
+  border-color: white;
+}
+.btn-primary.dropdown-toggle {
+    color: #fff;
+    border-color: #fff;
+}
+.bootstrap-select .dropdown-toggle:focus {
+  outline: none!important;
+  box-shadow:
```

Comment:
On this selector and the next one the value of `box-shadow` property is on a new line, unlike on line 50.
Let's keep it on the same line each time.

Reply:
Fixed